app/lib/package/backend.dart

```diff
@@ -1575,6 +1575,24 @@ class PackageBackend {
       current.gcpLock = GcpPublishingLock(oauthUserId: agent.payload.sub);
     }
   }
+
+  // ignore: deprecated_member_use_from_same_package
+  /// Migrates the deprecated [Package.automatedPublishingJson] JSON field
+  /// and makes sure that the only related value is in the
+  /// [Package.automatedPublishingField].
+  Future<void> backfillAutomatedPublishingField() async {
+    await for (final p in db.query<Package>().run()) {
+      // ignore: deprecated_member_use_from_same_package
+      if (p.automatedPublishingJson == null) {
+        continue;
```

Comment:
Can there be cases where both fields are non-null?
Should we avoid updating in that case?

Reply:
There may be such cases, however, we already ignore one of them. Better to consolidate on the field that is being used anyway.